helpers/addresses.py

```diff
@@ -789,7 +792,9 @@
     "spookyswap": {
         "router": "0xF491e7B69E4244ad4002BC14e878a34207E38c29",
         "factory": "0x152eE697f2E276fA89E96742e9bB9aB1F2E61bE3"
-    }
+    },
+    "registry": "0xFda7eB6f8b7a9e9fCFd348042ae675d1d652454f",
+    "registryV2": "0xdc602965F3e5f1e7BAf2446d5564b407d5113A06",
```

Comment:
getting a diff here with the registry v2 contract as it can be found here: https://github.com/jack-the-pug/badger-registry/blob/2.0/contracts/BadgerRegistry.sol

https://www.diffchecker.com/eIaJs6W3

all other addresses check out.